lab/lab2/js/part2-app-state.js

```diff
@@ -34,10 +34,23 @@
 ===================== */
 
 // We set this to HTTP to prevent 'CORS' issues
-var downloadData = $.ajax("http://");
-var parseData = function() {};
-var makeMarkers = function() {};
-var plotMarkers = function() {};
+var downloadData = $.ajax("https://raw.githubusercontent.com/CPLN692-MUSA611/datasets/master/json/philadelphia-crime-snippet.json");
+var parseData = function(data) {
+  return JSON.parse(data)
+  };
+var makeMarkers = function(parseData) {
+  var markers = [];
+  _.forEach(parseData, function(x){
```

Comment:
Small improvement suggestion: using `_.map` instead of `_.each`, which creates the array and doesn't require the `push`